---
title: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
event_type: "ELIXIR Training Course"
type: training_courses_2025
description: 10-12 February 2025 <br> Bari, Italy
state: closed
event:
  - name: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
    organisers: R. Calogero <br> E. Picardi <br> F. De Leo <br> A. Via <br> L. Le Pera <br> G. Pesole
    venue: Bari, Italy
    event_date: 2025/02/10-12
    state: closed

short_description: "This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package. \n"
teachers: [Luca Alessandrì, Raffaele Calogero]
keywords: "scRNA-seq, rCASC package, clustering methods, Genomics, Bioinformatics"
courseMode: onsite
addressLocality: Bari
addressCountry: Italy
startDate: "2025/02/10"
endDate: "2025/02/12"
duration: P3DT7H
inLanguage: en-EN 






short_description: "Description of the course"
teachers: [First Name1 Last Name1,First Name2 Last Name2]
keywords: "scRNA-seq, Proteomics, Genomics, Bioinformatics"
courseMode: onsite/online
addressLocality: City
addressCountry: Country
startDate: "2025/02/10"
endDate: "2025/02/12"
duration: P3DT7H
inLanguage: en-EN 











---


![Banner](images/2025_events/Banner_loghi_scRNA.png){: height="1200px" width="1200px"}
![logo_IBIOM](images/2025_events/logo_IBIOM.png){: height="100px" width="100px"}
![logo_uniba](images/2025_events/logo_uniba.png){: height="100px" width="100px"}
<br>
<br>

## Course Description

This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package [https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC). Participants will delve into experimental design principles, quality control for single-cell sequence outputs, and data reduction and clustering techniques to identify cell subpopulations. The course also offers an overview of the computational theory behind scRNA-seq tools, empowering participants to make informed decisions during analysis. Practical, hands-on sessions will allow attendees to work with real datasets and integrate results to extract meaningful biological insights.


<br>
<br>

## Important Dates

- Deadline for applications: **20th January 2025**
- Chosen participants will be notified by: 27th January 2025  
- Course date: **10-12 February 2025**


<br>
<br>

### Venue

University of Bari  
Aula 2, Old Biology Department, University Campus,  
University of Bari Aldo Moro [google maps link](https://maps.app.goo.gl/jsjYKkhYoARdYTtr6)<br>
Via Orabona 4
70125 Bari, Italy

<br>
<br>

### Fee 

The course includes a fee of **150 Euros for academic attendees** and **400 Euros for industry professionals**. Participants are expected to pay their own travel and accommodation costs (if any).

<br>
<br>

### Selection  

A maximum of **20 candidates** will be selected on a **first-come- first-served basis**. Selected participants will be notified by **27th January 2025**.

<br>
<br>

### Instructors {#instructors-section}


- Raffaele Calogero, University of Turin, Italy  
- Luca Alessandrì, University of Turin, Italy

<br>
<br>

### Helpers

- Ernesto Picardi, University of Bari, Italy  
- Adriano Fonzino, University of Bari, Italy  
- Pietro D’Addabbo, University of Bari, Italy 

<br>
<br>

### Scientific/Organising committee

- Raffaele Calogero, University of Turin, Italy  
- Ernesto Picardi, University of Bari, Italy  
- Loredana Le Pera, ISS, Rome, Italy (ELIXIR-IT Training Platform)  
- Allegra Via, Sapienza University, Rome, Italy (ELIXIR-IT Training Platform)
- Francesca De Leo, CNR, Bari, Italy  
- Graziano Pesole, University of Bari, Italy

<br>
<br>

### Organising secretary
  
- Marilena D’Ambrosio, CNR, Bari, Italy  
- Chiara Bruno, CNR, Italy
- Gianmarco Pascarella - CNR, Italy (ELIXIR-IT Training Platform)
- Irene Artuso, ISS, Rome, Italy (ELIXIR-IT Training Platform)

<br>

## Contact 

For all kinds of queries, please contact the Local Organizers at: [elixir.it.iib@gmail.com](mailto:elixir.it.iib@gmail.com) 

<br>
<br>

### Target audience

A maximum of 20 candidates will be selected. The workshop is suitable for life scientists who are new to single cell gene expression technology data analysis. It is open to PhD and Post-Doc research scientists, as well as team leaders and PIs. No prior knowledge of statistics or computing skills is required. However, a basic understanding of R programming is preferred. Prior knowledge of single-cell sequencing technologies is required. 

<br>
<br>

### Aims of the workshop 

- Introduce the principles and importance of designing effective scRNA-seq experiments.   
- Guide participants through quality control, data reduction, and clustering methods for scRNA-seq data.   
- Provide hands-on experience to develop practical analysis skills and enable the extraction and interpretation of biological insights from scRNA-seq datasets.

<br>
<br>

### Resources and tools covered

rCASC package for scRNA-seq analysis ([https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC))

<br>
<br>

### Learning Outcomes

By the end of this course, participants will be able to:

* Describe the importance of experimental design to ask sensible biological questions at single cell level.  
* Assess the quality of your data.  
* Describe limits and strength of clustering in scRNA-seq.  
* Identify genes driving cluster formation in scRNA-seq.  
* Annotate cell types  
* Performe batch correction and experiment integration on scRNA-seq data.

<br>
<br>

### Course prerequisites

Before starting the course, participants are requested to follow a brief and simple course on R, approximately 4 hours. The video lessons and lesson pdfs with exercises are available at [https://bit.ly/47XmDwx](https://bit.ly/47XmDwx) 

<br>
<br>

## Registration

[Application Form](https://forms.gle/Aj9VaxdkBhwFEZnB8)

<br>
<br>

## Programme

<table border="1" width="700">
  <tr>
    <td colspan="4" align=center><h3>10 February</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>Time</td>
    <td height="50" align=center>Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Experimental design <br>Reproducibility<br>Dense and sparse matrices<br>Data structure 10xgenomics, visium, curio bioscience</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="4" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">QC<br>Data imputation/normalisation<br>Exercises</td>
  </tr>
  <tr>
    <td colspan="4" align=center><h3>11 February</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Dimensionality reduction<br>Clustering<br>Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="4" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Cluster's specific genes extraction<br>Exercises</td>
  </tr>
  <tr>
    <td colspan="4" align=center><h3>12 February</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Cell type annotation<br>Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="4" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Data integration<br>Exercises</td>
  </tr>
