---
title: "Computational Methods for Epitranscriptomics"
event_type:  "ELIXIR Training Course"
type: training_courses
description: 11-13 September 2024 <br> Bari, Italy
event:
  - name: "Computational Methods for Epitranscriptomics"
    organisers: E. Picardi <br> F. De Leo <br> F. Nicassio <br> G. Pesole
    venue: Roma, Italy
    event_date: 2024/09/11-13
    state: open

short_description: "This training course will provide a foundation on the experimental planning, analytical strategies, and computational approaches for applying Nanopore direct RNA Sequencing to profile RNA modifications."
teachers: [Adriano Fonzino, Alessandro Silvestris, Francesco Nicassio, Ernesto Picardi, Tommaso Leonardi, Camilla Ugolini, Logan Mulroney]
keywords: "Bioinformatics, Epitranscriptomics, REDItools, REDIportal"
courseMode: onsite
addressLocality: Bari
addressCountry: Italy
startDate: "2024/09/11"
endDate: "2024/09/13"
duration: P3DT19H
inLanguage: it-IT   
---

![epitrascriptomics_bari](images/2024_events/epitrascriptomics_bari.png)


<br>

## Course Description

Epitranscriptome refers to all chemical modifications affecting cellular RNAs. Although more than 170 different types have been identified up to now, their precise detection remains challenging. Profiling epitranscriptome modifications require ad hoc sequencing technologies and dedicated computational workflows. While short-read RNA sequencing (e.g., Illumina) is the de facto method for unveiling RNA editing events (rather than transient modifications), direct RNA Sequencing by Oxford Nanopore Technology has recently become one of the most powerful techniques to profile the full repertoire of RNA modifications. Its capacity to directly sequence full-length, native RNA molecules without the need for reverse transcription or PCR amplification offers remarkable advantages compared to other short-read sequencing technologies. However, it also presents various experimental and analytical challenges.

This training course will provide a foundation on the experimental planning, analytical strategies, and computational approaches for applying Nanopore direct RNA Sequencing to profile RNA modifications. Through a combination of theoretical lectures and practical exercises, participants will learn the fundamental concepts and bioinformatics skills that will allow them to start using Nanopore sequencing for epitranscriptome analysis. Additionally, the course will also cover theoretical aspects of RNA editing as well as practical sessions focused on its detection in human/mouse transcriptomes by Illumina RNAseq data. Participants will acquire specific skills in command-line tools (REDItools) to call, annotate (REDIportal), and filter RNA variants.

You can find here the complete [program](https://docs.google.com/document/d/1dxu3OFrOvllSAV36AZqIyslyePhazyST/edit). 

<br>

## Important Dates

Deadline for applications: **22 July 2024** <br>
Course date: **11-13 September 2024** <br>

<br>

### Venue

Aula 4 - Vecchi Istituti Biologici<br>
Campus Universitario “E. Quagliariello”<br>
Università degli Studi di Bari<br>
Via Orabona 4<br>
70125 Bari, Italy<br>
[We are here](https://maps.app.goo.gl/VV2TKGsEBWaTTCqX9)

<br>

### Fee

The course fee is 100 euros, covering access to computing infrastructure (for the 3-day course) training material, lunches, and coffee breaks.

<br>

### Selection
A maximum of **30** candidates will be accepted. Course selection will be done according to the needs of the course as emerging from the application [form](https://docs.google.com/forms/d/e/1FAIpQLSco6bmrpJPmFrIeQxjuAIsyWQgXF2WjO6nPyvL1zOkNwPupNA/viewform). Priority will be given to candidates from ELIXIR-IT member institutions (see the [list](https://elixir-italy.org/about/)), ELIXIR nodes as well as candidates involved in the following PNRR programs: CN3-RNA, CN1-HPC, AgetIt, and Mnesys. 
Acceptance and payment instructions will be notified soon after the application deadline.

<br>

## Application Form
[Application Form - Registration](https://docs.google.com/forms/d/e/1FAIpQLSco6bmrpJPmFrIeQxjuAIsyWQgXF2WjO6nPyvL1zOkNwPupNA/viewform)

<br>

### Organisers
- Ernesto Picardi (UNIBA, IT)
- Francesca De Leo (CNR-IBIOM, IT)
- Francesco Nicassio (IIT, IT)
- Graziano Pesole (UNIBA, IT)

<br>

### Instructors
- Adriano Fonzino (UNIBA, IT)
- Alessandro Silvestris (UNIBA, IT)
- Francesco Nicassio (IIT, IT)
- Ernesto Picardi (UNIBA, IT)
- Tommaso Leonardi (IIT, IT)
- Camilla Ugolini (IIT, IT)
- Logan Mulroney (IIT, IT)


<br>

### Helpers
- Chiara Bruno (CNR, IT)
- Marilena D’Ambrosio (CNR, IT)
- Pietro D’Addabbo (UNIBA, IT)


<br>

## Contact
Should you have any questions, do not hesitate to contact the course Local Organizers ([elixirit.tp.bari@gmail.com](mailto:elixirit.tp.bari@gmail.com)).

<br>

### Target audience
The theoretical lectures are intended for early/mid-career stage researchers with a solid understanding of molecular biology and RNA sequencing, but no prior experience with Nanopore sequencing is required. To follow and independently carry out the practical exercises participants will need basic bioinformatics skills and a certain degree of familiarity using command-line tools in a *nix environment* (e.g. Linux or MacOS). 

<br>

### Learning Outcomes
By the end of this course, the participants will be able to:

- acquire theoretical knowledge about methods to profile RNA modifications
- use bioinformatics tools to analyze Nanopore reads
- use bioinformatics tools to pre-process RNAseq data
- use command-line tools (REDItools) for calling, annotating (REDIportal), and filtering RNA variants

<br>

### Laptop
Participants are kindly requested to bring their laptops. In case of need, organizers will provide participants with a computer.

<br>
