---
title: "Computational Pangenomics"
event_type:  "ELIXIR Training Course"
type: training_courses
description: 19-21 June 2024 <br> Roma, Italy
state: closed
event:
  - name: "Computational Pangenomics"
    organisers: V. Colonna <br> E. Garrison <br> A. Guarracino <br> L. Licata <br> F. Villani 
    venue: Roma, Italy
    event_date: 2024/06/19-21
    state: open

short_description: "Participants will learn about pangenome concepts and gain practical <br>experience building and analyzing pangenome graphs. They will apply these methods to complex <br>research questions that require understanding the relationships between multiple genomes or <br>accounting for variability when analyzing new genomes. By the end of the practical course / <br>workshop, participants will have a strong understanding of pangenome methods based on whole <br>genome assemblies."
teachers: [Vincenza Colonna, Erik Garrison, Andrea Guarracino, Luana Licata, Flavia Villani]
keywords: "Pangenomics, Genome, Whole Genome Assemblies, bioinformatics."
courseMode: onsite
addressLocality: Rome
addressCountry: Italy
startDate: "2024/06/19"
endDate: "2024/06/21"
duration: P3DT19H
inLanguage: it-IT   
---

<br>

![Logo_IIB](images/infrastructures/logo_iib.png) ![Logo_Bicocca](images/infrastructures/logo_bicocca.png)![Elixir_Excelerate](images/infrastructures/Logo_excelerate_whitebackground.png){: height="200px" width="200px"} <br>
![Elixir_Sweden](images/infrastructures/Logo_elixir_sweden.png){: height="200px" width="200px"} ![Elixir_France](images/infrastructures/Logo_elixir_france.png){: height="200px" width="200px"} ![Elixir_Slovenia](images/infrastructures/Logo_ELIXIR_SLOVENIA_white_background.png){: height="200px" width="200px"} <br>
![Logo_INRA](images/infrastructures/Logo_INRA.png){: height="200px" width="200px"} ![Logo_CNR](images/infrastructures/Logo_CNR_Italy.png){: height="200px" width="200px"} ![logo_NBIS](images/infrastructures/Logo_NBIS.png){: height="200px" width="200px"} 

<br>
<br>

## Course Description
Participants will learn about pangenome concepts and gain practical experience building and analyzing pangenome graphs. They will apply these methods to complex research questions that require understanding the relationships between multiple genomes or accounting for variability when analyzing new genomes. By the end of the practical course / workshop, participants will have a strong understanding of pangenome methods based on whole genome assemblies.
<br>
<br>

## Important Dates 

Deadline for applications: **24 May 2024** <br>
Chosen participants will be notified by: **31 May 2024** <br>
Course date: **19-21 June 2024** <br>
<br>
<br>

### Venue

Aula informatizzata, 2nd Floor, Building CU010, Sapienza University of Rome, Piazzale Aldo Moro 5, 00185 Rome, Italy **[link to the map](https://maps.app.goo.gl/REeKpiBCU8PzoorZ8)**
<br>
<br>

### Fee

The course includes a fee of 70â‚¬ which covers coffee-breaks. Participants are expected to pay their own **travel** and **accomodation** costs (if any).
<br>
<br>

### Selection
A maximum of **24** candidates will be selected based on their need for the course as emerging from the application form. Chosen participants will be notified by 17th May 2024. Priority will be given to candidates from ELIXIR-IIB member institutes (see the list at the bottom) and ELIXIR Nodes.
Go to the Application Form - Registration.

**Cancellation policy**: Attendance is limited to 24 participants. We expect many more applications. Accepted participants commit to attend the course for its whole duration. Failure to attend training sessions is disruptive. Moreover, it blocks other candidates from participating. Therefore, a cancellation policy is in place so that only written requests presented 10 days in advance relatively to the course starting date are accepted.
<br>
<br>

### Instructors
- Erik Garrison - University of Tennesse Health Science Center
- Andrea Guarracino - University of Tennesse Health Science Center
<br>
<br>

### Helpers
- Flavia Villani - University of Tennesse Health Science Center
<br>
<br>

### Organisers
- Vincenza Colonna - University of Tennesse Health Science Center
- Erik Garrison - University of Tennesse Health Science Center
- Andrea Guarracino - University of Tennesse Health Science Center
- Luana Licata Tor Vergata University of Rome - ELIXIR-IT
- Flavia Villani - University of Tennesse Health Science Center
<br>
<br>

## Contact
For all kinds of queries, please contact the local organiser Luana Licata at: [luana.licata@gmail.com](mailto:luana.licata@gmail.com). For questions about course content, methods, and materials, please contact Erik Garrison [egarris5@uthsc.edu](mailto:egarris5@uthsc.edu) and Andrea Guarracino [aguarra1@uthsc.edu](mailto:aguarra1@uthsc.edu).
<br>
<br>

### Target audience
This course is aimed at PhD-students, post docs or other researchers interested in learning about pangenomics. It will be especially useful for researchers that have (or will have) data of their own.
<br>
<br>

### Learning outcomes
By the end of this course, learners will be independently able to:

- Understand the concepts and importance of pangenomics in addressing reference bias and capturing genome variability.
- Apply pangenome methods to complex research questions that require understanding the relationships between multiple genomes or accounting for variability when analyzing new genomes.
- Gain practical experience in building pangenome graphs from whole-genome assemblies using tools like PGGB and ODGI.
- Learn how to analyze and interpret pangenome graphs
<br>
<br>

### Course prerequisites
The participants will need to have some experience in Linux command line (bash), enough to start tools and find paths to files. Experience with Next Generation Sequencing data and common file formats (fasta, fastq, bam, etc.) will be extremely useful, but is not mandatory.
<br>
<br>

## Registration
[Application Form](https://docs.google.com/forms/d/e/1FAIpQLScYGd0K2ZUD4sZne4XAZbfYV7tJhdYluAB7Z1lr8VNOqU_cKg/viewform)

After selection, confirmed participants will be able to access the course agenda, materials and communication tools on the [ELIXIR-SI eLearning Platform (EeLP)](https://elixir.mf.uni-lj.si/enrol/index.php?id=45).
<br>
<br>

## Programme

<table border="1" width="700">
  <tr>
    <td colspan="4" align=center><h3> Wednesday - 19 June</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 09:00 - 10:00</td>
    <td height="50" align=center> Lecture</td>
    <td height="50">Erik Garrison</td>
    <td height="50">Introduction to pangenomics</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:00 - 10:30</td>
    <td colspan="3" height="50" align=center> Break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:30 - 12:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>
    <td height="50">HLA & LPA graph building</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 12:00 - 13:30</td>
    <td colspan="3" height="50" align=center>Lunch break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30 - 14:00</td>
    <td height="50" align=center> Lecture</td>
    <td height="50">Andrea Guarracino</td>    
    <td height="50">Understanding pangenomes</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>14:00 - 15:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">Human pangenome graph building</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 15:00 - 15:30</td>
    <td colspan="3" height="50" align=center> Break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:30 - 17:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">ODGI applications</td>
  </tr>
  <tr>
    <td colspan="4" align=center><h3>Thursday - 20 june</h3></td>
  </tr>
  <tr>
    <td height="50" width="100">09:00 - 10:00</td>
    <td height="50" align=center>Lecture + Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">Discussion about previous day activities</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:00 - 10:30</td>
    <td colspan="3" height="50" align=center> Break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:30 - 12:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>
    <td height="50">Yeast pangenome graph building</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 12:00 - 13:30</td>
    <td colspan="3" height="50" align=center>Lunch break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30 - 14:00</td>
    <td height="50" align=center> Lecture</td>
    <td height="50">Andrea Guarracino</td>    
    <td height="50">Long-read Assembly</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>14:00 - 15:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">Assembly and QC</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 15:00 - 15:30</td>
    <td colspan="3" height="50" align=center> Break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:30 - 17:00</td>
    <td height="50" align=center> Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">Assembly and pangenome graphs</td>
  </tr>
  <tr>
    <td colspan="4" align=center><h3>Friday - 21 june</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:00 - 10:00</td>
    <td height="50" align=center>Lecture + Practical</td>
    <td height="50">Everyone</td>    
    <td height="50">Discussion about previous day activities</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:00 - 10:30</td>
    <td colspan="3" height="50" align=center> Break</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center> 10:30 - 12:00</td>
    <td colspan="3" height="50" align=center>Wrap-up and feedback session</td>
  </tr>
</table>




### ELIXIR-IIB member institutes
1. CNR, National Research Council (Lead Institute)
2. CRS4
3. CINECA
4. Edmund Mach Foundation, Trento
5. ENEA
6. Fondazione Telethon
7. INFN
8. Istituto Superiore di SanitÃ  (ISS)
9. GARR
10. Stazione Zoologica Anton Dohrn, Napoli
11. University of Roma "Sapienza"
12. University of Bari
13. University of Bologna
14. University of Firenze
15. University of Milano
16. University of Milano Bicocca
17. University of Napoli
18. University of Padova
19. University of Parma
20. University of Roma "Tor Vergata"
21. University of Salerno
22. University of Torino
23. University of Tuscia