---
title: "School of Python for genomics. Intermediate module: Analysing your genomics data with Python"
description: 30 January - 2 February 2024 <br> Roma, Italy
page_id: python_school_2024
event:
  - name: "School of Python for genomics. Intermediate module: Analysing your genomics data with Python"
    organisers: L. Le Pera <br> S. Morabito <br> A. Via <br> SIDiLV <br> ELIXIR-IT 
    teachers: []
    venue: Roma, Italy
    event_date: 2024/01/30
    state: open
---

<!-- [{% include image.html file="/2022_and_older/elixir_banner_22novembre_blu.png" alt="elixir_banner_22novembre_blu" max-width="10" %}](https://bioinfo.medils.hr/) -->
<br>
<br>

## Important Dates 
- Deadline for applications: **8 January, 2024**
- School date: **30 January - 2 February, 2024** 
<br>
<br>

### Venue
Sapienza University of Rome<br>
P.le Aldo Moro, 5<br>
00195 Rome, Italy
<br>
<br>

### Fee
No registration fee is required, but those who are selected will be expected to pay the [SIDiLV](https://sidilv.org/) membership fee for the year 2024 afterward (Full membership fee: 50 euros; Reduced membership fee for individuals under 35 years of age: 30 euros).<br>
*Participants are expected to pay their travel, meal, and hotel costs (if any).
<br>
<br>

### Registration 
## [Application Form](https://forms.gle/BUme7erea4nSyeBSA)
<br>

A maximum of **16** candidates will be selected based on a first-come, first-served basis, and the pre-requisites requested for attending the course, as emerging from the application form. 
Notifications of acceptance will be sent by *January 10th*. 
<br>
<br>

### Instructors
- **Luca De Sabato** - Dep. of Food Safety, Nutrition and Veterinary Public Health, National Institute of Health (SANV - ISS), Italy
- **Arnold Knijn** - Dep. of Food Safety, Nutrition and Veterinary Public Health, National Institute of Health (SANV - ISS), Italy
- **Loredana Le Pera** - Core Facilities Technical-Scientific Service, National Institute of Health (FAST - ISS), Italy
- **Allegra Via** - Sapienza University of Rome, Italy
<br>
<br>

### Helpers
- **Gianluca Frustagli** - Core Facilities Technical-Scientific Service, National Institute of Health (FAST - ISS), Italy
- **Chiara Pacelli** - Dep. of Biochemical Sciences, Sapienza University of Rome, Italy
- **Irene Ruspantini** - Core Facilities Technical-Scientific Service, National Institute of Health (FAST - ISS), Italy
<br>
<br>

### Course Description
The use of Next-Generation Sequencing (NGS) has exponentially increased in the last two decades, both in the field of research and diagnostics. This technology allows obtaining millions of sequences ('reads') in a single experiment, making it possible to reconstruct complete genomes of pathogens relevant to public health in just a few hours.
By that motivation, the *[Societ√† Italiana di Diagnostica di Laboratorio Veterinaria (SIDiLV)](https://sidilv.org/)* and the *[Italian Infrastructure of Bioinformatics (IIB/ELIXIR Italy)](https://elixir-italy.org/about/)* organised the **School of Python for genomics** at the *[Italian National Institute of Health (ISS)](https://www.iss.it/en/web/guest/home)*.<br> 
The School aims to provide the necessary skills to utilise the Python language both for the use and development of software and for genomics data analysis in general.<be>

The School consists of 3 consequential modules:<br>
- [*Basic module*: Introduction to Python programming (Nov 2022)](https://elixir-iib-training.github.io/site/2022-11-22-School_Python_genomics_Basic_module)
- ****Intermediate module*: Analysing your genomic data with Python***
- *Hackathon module*: Python programming for developers

The main objective of this second Module in the 'School of Python for Genomics' titled 'Intermediate Module: Analysing your genomics data with Python' 
will be to characterise a population of Escherichia coli strains through the analysis of Whole Genome Sequencing (WGS) data, identifying any groups of isolates that share epidemiological and/or genomic characteristics and the presence of specific virulence traits.

This module aims to achieve the following objectives:

Identify the distribution of serogroups of isolates.
1. Analyse the presence and distribution of antibiotic-resistance genes.
2. Examine the presence and distribution of virulence genes.
3. Perform isolate typing (using methods such as MLST and cgMLST) and identify any correlations.

The course involves the development of a Python pipeline capable of processing raw data obtained from Illumina and/or Ion Torrent NGS.
<br>
<br>

### Prerequisites
This Intermediate Module is designed as a follow-up to the [Basic Module of the School of Python](https://elixir-iib-training.github.io/site/2022-11-22-School_Python_genomics_Basic_module), hence it assumes a basic understanding of programming and prior programming experience in Python as prerequisites. This Module will primarily focus on practical applications rather than theoretical concepts.

### Target audience
Research scientists at any stage of their career who are using or planning to use genomics data in their research activities are welcome to attend the workshop. <br> 
<br> 
<br>

### Learning Outcomes
At the end of the course, the student will be able to:
- Critically assess the quality of NGS data.
- Independently develop a complete pipeline for genomic analysis, starting from raw data to obtain final results.

In general, after completing the entire module, the student will have a greater awareness of both the use of Python and the impact that genomic analysis can have in a professional and social context.
<br> 
<br>

### Draft Program
<table border="1" width="700">
<tr>
   <td colspan="2"><h3>Tuesady - 30 January</h3></td>
</tr>
<tr>
   <td height="50" width="50">14:00 - 14:30</td>
   <td height="50">Welcome/Introduction(s)</td>
</tr>
<tr>
   <td height="50" width="50">14:30 - 16:00</td>
   <td height="50">Introduction to Linux shell</td>
</tr>
<tr>
   <td height="50" width="50">16:00 - 16:30</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">16:30 - 17:30</td>
   <td height="50">Introduction to the Python programming language</td>
</tr>
<tr>
   <td colspan="2"><h3>Wednesday - 31 January</h3></td>
</tr>
<tr>
   <td height="50" width="50">09:00 - 10:30</td>
   <td height="50">Introduction to Linux shell</td>
</tr>
<tr>
   <td height="50" width="50">10:30 - 11:00</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">11:00 - 12:30</td>
   <td height="50">Introduction to Python programming</td>
</tr>
<tr>
   <td height="50" width="50">12:30 - 14:00</td>
   <td colspan="1">Lunch</td>
</tr>
<tr>
   <td height="50" width="50">14:00 - 15:30</td>
   <td height="50">Introduction to Python programming</td>
</tr>
<tr>
   <td height="50" width="50">15:30 - 16:00</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">16:00 - 17:30</td>
   <td height="50">Introduction to Python programming</td>
</tr>
<tr>
   <td colspan="2"><h3>Thursday - 1 February</h3></td>
</tr>
<tr>
   <td height="50" width="50">09:00 - 10:30</td>
   <td height="50">Introduction to Python programming</td>
</tr>
<tr>
   <td height="50" width="50">10:30 - 11:00</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">11:00 - 12:30</td>
   <td height="50">Introduction to Python classes and libraries</td>
</tr>
<tr>
   <td height="50" width="50">12:30 - 14:00</td>
   <td colspan="1">Lunch</td>
</tr>
<tr>
   <td height="50" width="50">14:00 - 15:30</td>
   <td height="50">Biopython and pipelines (genomics applications)</td>
</tr>
<tr>
   <td height="50" width="50">15:30 - 16:00</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">16:00 - 17:30</td>
   <td height="50">Biopython and pipelines (genomics applications)</td>
</tr>
<tr>
   <td colspan="2"><h3>Friday - 2 February</h3></td>
</tr>
<tr>
   <td height="50" width="50">09:00 - 10:30</td>
   <td height="50">Putting everything together: final exercise</td>
</tr>
<tr>
   <td height="50" width="50">10:30 - 11:00</td>
   <td colspan="1">Break</td>
</tr>
<tr>
   <td height="50" width="50">11:00 - 12:30</td>
   <td height="50">Putting everything together: final exercise</td>
</tr>
<tr>
   <td height="50" width="50">12:30 - 13:30</td>
   <td height="50">Wrap-up and feedback questionnaire</td>
</tr>
</table>
<br>
<br>

### Organisers
- **Stefano Morabito** - SIDiLV/ELIXIR Italy/National Institute of Health (SANV - ISS), Italy 
- **Allegra Via** - ELIXIR Italy (Training Platform)/Sapienza Univ. of Rome
- **Loredana Le Pera** - ELIXIR Italy (Training Platform)/National Institute of Health (FAST - ISS), Italy
<br>
<br>

### Contact
For all kinds of queries, contact us at <elixir.ita.training@gmail.com> 
<br>
<br>
<table>
<tr>
	<td>
	<a href="https://sidilv.org/"><img height="140" src="../../../img/Logo_SIDILV.png"></a>
	</td>
	<td weight="20"></td>
	<td>
	<a href="https://elixir-iib-training.github.io/website/"><img src="../../../img/logo_iib.png"></a>
	</td>
</tr> 
</table>
