---
title: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
event_type: "ELIXIR Training Course"
type: training_course
description: 25-28 November 2025 <br> Roma, Italy
event:
  - name: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
    organisers: R. Calogero <br> L. Le Pera <br> 
    venue: Rome, Italy
    event_date: 2025/11/25-28
    

short_description: "This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package."
teachers: [Luca Alessandrì, Raffaele Calogero]
keywords: "scRNA-seq, rCASC package, clustering methods, Genomics, Bioinformatics"
courseMode: onsite
addressLocality: Rome
addressCountry: Italy
startDate: "2025/11/25"
endDate: "2025/11/28"
duration: P3DT7H
inLanguage: en-EN 
---


![Banner](images/2025_events/barra_loghi_rome_technopole_Bianco.jpg){: height="1200px" width="1200px"}

<table border="0" align="center">
  <tr> 
    <td><a href=""><img src="images/2025_events/Logo_ISS.png" height="100"></a></td>
    <td width="10"></td>
	<td><a href=""><img src="images/2025_events/sco.png" height="130"></a></td>
    <td width="10"></td>
	<td><a href=""><img src="images/2025_events/BITSLogo.jpg" height="130"></a></td>
	<td width="10"></td>
    <td><a href=""><img src="images/2025_events/Logo_infolife.jpg" height="100"></a></td>
  </tr>
	<tr>
	<td height="10" width="10"></td>	
	</tr>
	<tr>
	<td><a href="http://elixir-italy.org"><img src="images/2025_events/IIB-ELIXIR_col.png" height="100" width="200"></a></td>
	</tr>
</table>

<br>
<br>

## Course Description

This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package [https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC). Participants will delve into experimental design principles, quality control for single-cell sequence outputs, and data reduction and clustering techniques to identify cell subpopulations. The course also offers an overview of the computational theory behind scRNA-seq tools, empowering participants to make informed decisions during analysis. Practical, hands-on sessions will allow attendees to work with real datasets and integrate results to extract meaningful biological insights. The workshop is organised in collaboration with the **ELIXIR-IT Compute Platform** (Data Center ReCaS-Bari), which provides the computational infrastructure and support for the hands-on sessions.<br>
This course is organised within the project “Rome Technopole Innovation Ecosystem”.
<br>

## Important Dates

- Deadline for applications: **15 Nov 2025**
- Course date: **25-28 November 2025**

<br>

### Venue

Sapienza University of Rome [google link](https://maps.app.goo.gl/MCnUBWRy8qeaQXqj9)
P.le Aldo Moro, 5
00195 Rome, Italy

<br>

### Fee 

The course does not include any fee. Participants are expected to pay their own travel and accomodation costs (if any).

<br>

### Selection  

A maximum of **15 participants** will be selected on a first-come, first-served basis. Selected participants will be notified progressively as registrations are processed.

<br>

### Instructors

- Raffaele Calogero, University of Turin, Italy  
- Luca Alessandrì, University of Turin, Italy
- Valentina Libri, ISS, Rome, Italy

<br>

### Scientific/Organising committee

- Raffaele Calogero, University of Turin, Italy
- Loredana Le Pera, ISS - FAST, Rome, Italy (ELIXIR-IT Training Platform)
- Allegra Via, Sapienza University, Rome, Italy (ELIXIR-IT Training Platform)
- Giacinto Donvito, INFN, Bari, Italy (ELIXIR-IT Compute Platform)
- Rino Ragno, Sapienza University, Rome, Italy (ELIXIR-IT LTeC)

<br>

### Organising secretary
  
- Gianmarco Pascarella - CNR, Italy (ELIXIR-IT Training Platform)
- Irene Artuso, ISS, Rome, Italy (ELIXIR-IT Training Platform)

<br>

## Contact 

For all kinds of queries, please contact the Local Organisers at: [elixir.it.iib@gmail.com](mailto:elixir.it.iib@gmail.com) 

<br>

### Target audience

A maximum of 15 candidates will be selected. This course is aimed at life science researchers, students, principal investigators (PIs), and professionals from industry interested in analysing single-cell RNA-seq data. No prior knowledge of statistics or advanced computing skills is required. However, **a basic understanding of R programming** and bioinformatics concepts is preferred. **Prior knowledge of single-cell sequencing technologies** is required.

<br>
<br>

### Aims of the workshop 

- Introduce the principles and importance of designing effective scRNA-seq experiments.   
- Guide participants through quality control, data reduction, and clustering methods for scRNA-seq data.   
- Provide hands-on experience to develop practical analysis skills and enable the extraction and interpretation of biological insights from scRNA-seq datasets.

<br>

### Resources and tools covered

rCASC package for scRNA-seq analysis ([https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC))

<br>

### Learning Outcomes

By the end of this course, participants will be able to:

* Describe the importance of experimental design to ask sensible biological questions at single cell level.  
* Assess the quality of your data.  
* Describe limits and strength of clustering in scRNA-seq.  
* Identify genes driving cluster formation in scRNA-seq.  
* Annotate cell types  
* Performe batch correction and experiment integration on scRNA-seq data.

<br>

### Course prerequisites

Before starting the course, participants are requested to follow a brief and simple course on R, approximately 4 hours. The video lessons and lesson pdfs with exercises are available at [https://bit.ly/47XmDwx](https://bit.ly/47XmDwx) 

<br>

## Registration

[Application Form]()

<br>

## Programme

<table border="1" width="700">
<tr>
    <td colspan="3" align=center><h3>25 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>Time</td>
    <td height="50" align=center>Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
 <tr>
    <td height="50" width="100" align=center>14:30-16:30</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Welcome and introduction to the workshop<br>Lecture: Experimental overview of single-cell RNA-seq<br>Setting up the working environment</td>
  </tr>
 
 <tr>
    <td colspan="3" align=center><h3>26 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>Time</td>
    <td height="50" align=center>Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Experimental design <br>Reproducibility<br>Dense and sparse matrices<br>Data structure 10xgenomics, visium, curio bioscience</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">QC<br>Data imputation/normalisation<br>Exercises</td>
  </tr>
  <tr>
    <td colspan="3" align=center><h3>27 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Dimensionality reduction<br>Clustering<br>Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Cluster's specific genes extraction<br>Exercises</td>
  </tr>
  <tr>
    <td colspan="3" align=center><h3>28 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Cell type annotation<br>Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=left><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-16:30</td>
    <td height="50" align=center>Lecture/Exercise</td>
    <td height="50">Data integration<br>Exercises</td>
  </tr>
</table>
