---
title: "Profiling of microbial communities using targeted and shotgun metagenomics"
event_type: "ELIXIR Training Course"
type: training_courses
description: 30 June - 4 July 2025 <br> Bari, Italy

event:
  - name: "Profiling of microbial communities using targeted and shotgun metagenomics"
    organisers: G.Defazio<br> C.Donati<br> B.Fosso<br> P.Manghi<br> M.Marzano<br> P.Novielli<br> G.Pesole<br> M.Santamaria<br> S.Tangaro
    venue: Bari, Italy
    event_date: 2025/05/25-30
    state: open

short_description: "This training course focuses on the study of the microbiota using Next Generation Sequencing (NGS) techniques. The course will introduce DNA metabarcoding and shotgun metagenomics and illustrate the major computational tools for the analysis of metagenomic data. In addition, the course will provide an introduction to Machine Learning methods applied to the analysis of metagenomic data. The course will include both a theoretical introduction to the topics and practical sessions with real data."
teachers: [Duccio Cavalieri, Eugenio Parente, Antonella Bruno]
keywords: "Metagenomics, DNA metabarcoding, Next Generation Sequencing (NGS), Machine Learning, microbial, metabolomics"
courseMode: onsite
addressLocality: Bari
addressCountry: Italy
startDate: "2025/06/30"
endDate: "2025/06/4"
duration: P5DT35H
inLanguage: it-IT   
---
<br>
<br>

<!--
![Banner](images/funders/Banner_CNR.png){: height="1200px" width="1200px"}
-->
<br>
<br>

## Course Description 

This training course focuses on the study of the microbiota using Next Generation Sequencing (NGS) techniques. The course will introduce DNA metabarcoding and shotgun metagenomics and illustrate the major computational tools for the analysis of metagenomic data. In addition, the course will provide an introduction to Machine Learning methods applied to the analysis of metagenomic data. The course will include both a theoretical introduction to the topics and practical sessions with real data.


<br>
<br>

## Important Dates

- Deadline for applications: **20th May 2025**
- Chosen participants will be notified by: **31st May 2025**
- Course date: **30 June - 4 July 2025**

<br>
<br>

### Venue

University of Bari “Aldo Moro”
Aula 4, [“Vecchi Istituti Biologici”](https://maps.app.goo.gl/8vMErM9DugHk55QGA), Campus Ernesto Quagliarello, via  Orabona 4, 70126, Bari.
The closest Campus entrance is on Via Giovanni Amendola 165/A

<br>
<br>

### Fee

The course includes a fee of 250 Euros for academic attendees and 350 Euros for industry professionals, covering lunches and coffee breaks. Participants are expected to pay their own travel and accommodation costs (if any).

<br>
<br>

### Selection procedure

A maximum of 25 candidates will be selected on a first-come first-served basis. Selected participants will be notified by 31st May 2025.

<br>
<br>

### (Invited) Speakers

- Duccio Cavalieri, University of Florence
- Eugenio Parente, University of Basilicata
- Antonella Bruno, University of Milano Bicocca

<br>
<br>

### Organizers and Instructors

- Giuseppe Defazio, University of Bari “Aldo Moro”, ELIXIR-IT, Italy 
- Claudio Donati, Fondazione Edmund Mach, ELIXIR-IT, Italy
- Bruno Fosso, University of Bari “Aldo Moro”, ELIXIR-IT, Italy 
- Paolo Manghi, Fondazione Edmund Mach, ELIXIR-IT, Italy
- Marinella Marzano, Institute of Biomembranes, Bioenergetics and Molecular Biotechnologies, IBIOM-CNR, ELIXIR-IT, Italy
- Pierfrancesco Novielli, INFN-Bari, ELIXIR-IT, Italy
- Graziano Pesole, University of Bari “Aldo Moro”,  ELIXIR-IT, Italy
- Monica Santamaria, University of Bari “Aldo Moro”,  ELIXIR-IT, Italy
- Sabina Tangaro, University of Bari “Aldo Moro”, ELIXIR-IT, Italy 

<br>
<br>

## Contacts

For all kinds of queries, please contact Claudio Donati at: [claudio.donati@fmach.it](mailto:claudio.donati@fmach.it), Bruno Fosso at: [bruno.fosso@uniba.it](mailto:bruno.fosso@uniba.it) or Monica Santamaria: at [monica.santamaria@uniba.it](mailto:monica.santamaria@uniba.it)

<br>
<br>

### Target Audience

A maximum of **25 candidates** will be selected on a first-come- first-served basis. Selected participants will be notified by **31st May 2025**.
The workshop is open to all biological or biomedical PhD and Post-Doc research scientists. The course requires basic knowledge of Unix and the command line (bash shell). 

<br>
<br>

### Objectives

Gain familiarity with key metabolomics applications in fields such as environmental science, nutraceuticals, food science, and personalized medicine.
Learn the most appropriate techniques for sample preparation and data acquisition using mass spectrometry. 
Understand untargeted, targeted, and labeling approaches. Develop statistical analysis skills. 
Use metabolomics data to explore biochemistry and metabolism. 
Receive guidance on designing personalized metabolomics experiments (covering areas such as primary and secondary metabolism, lipidomics, food composition, natural products, exposome, and volatilome) based on a case study of interest.

<br>
<br>

### Aims of the workshop
- Introduce the principles of metabarcoding and shotgun metagenomics
- Guide participants through all steps of data analysis, from quality controls to data reduction and visualization
- Provide hands-on experience to develop practical analysis skills and enable the extraction and interpretation of biological insights from metabarcoding and metagenomic data

<br>
<br>

### Resources and tools covered

- MetaPhlAn, StrainPhLan, HUMANN, Kraken2, Bracken, Metabat2, kMetaShot
- QIIME2, DADA2, BioMaS, ITSoneDB/ITSoneWB, GreenGenes2

<br>
<br>

### Learning Outcomes 

By the end of this course, the participants will be able to:
Use the common bioinformatics workflows for DNA-metabarcoding and shotgun metagenomics data analysis;
Analyse microbial communities using common ecological measures of biodiversity;
Apply methods of multivariate statistics and dimensionality reduction to microbiome data; 
Use machine learning approaches for microbiota data analysis.
In summary, the course provides a comprehensive overview of modern microbiota analysis techniques, from sample preparation to bioinformatics analysis and machine learning applications.

<br>
<br>

### Learning Outcomes

By the end of this course, the participants will be able to:
- Use the common bioinformatics workflows for DNA-metabarcoding and shotgun metagenomics data analysis;
- Analyse microbial communities using common ecological measures of biodiversity;
- Apply methods of multivariate statistics and dimensionality reduction to microbiome data; 
- Use machine learning approaches for microbiota data analysis.
In summary, the course provides a comprehensive overview of modern microbiota analysis techniques, from sample preparation to bioinformatics analysis and machine learning applications.

<br>
<br>

### Prerequisites

Participants are expected to have basic understanding of the UNIX shell. 

<br>
<br>

## Registration

[Application Form](https://forms.gle/KYxXZ5zVA92Y8oqU7)

<br>
<br>

## Programme

<table border="1" width="800">
  <tr>
    <td colspan="4" align="center"><h3>Day 1</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">Time</td>
    <td height="50">Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">10:00-12:00</td>
    <td colspan="4" align="center">Welcome and registration</td>

  </tr>
  <tr>
    <td height="50" width="100" align="center">12:00-14:00</td>
    <td colspan="4" align="center">Welcome cocktail</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">14:00-14:30</td>
    <td colspan="4" align="center">Welcome and short intro on ELIXIR<br>Introduction to the course</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">14:30-17:30</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      What is metagenomics - metabarcoding vs shotgun<br>
      Intro on sequencing technologies<br>
       - Second Generation Technologies<br>
       - Third Generation Technologies<br>
      Introduction to DNA-metabarcoding<br>
       - Historical notes<br>
       - Applications: from the gut microbiome to food traceability<br>
      Experimental design
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">17:30-18:00</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">Access to virtual machines and upload/download tests of files and folders</td>
  </tr>
  <tr>
    <td colspan="4" align="center"><h3>Day 2 - Analysis of microbiome biodiversity through DNA metabarcoding</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">Time</td>
    <td height="50">Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">09:00-09:45</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Amplicon sequencing<br>
      Variable regions vs full length
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">09:45-10:30</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">
      Characteristics of the raw sequencing data<br>
       - Data visualization<br>
       - Data quality: fastqc/multiqc<br>
       - Data import into qiime<br>
       - Data pre-processing
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">10:30-11:00</td>
    <td colspan="4" align="left"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">11:00-12:00</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Denoising vs OTU-clustering & Chimera removal<br>
      Taxonomic classification: Approaches based on similarity analysis vs Bayesian classifiers
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">12:00-13:00</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">
      - Data denoising<br>
      - Taxonomic classification of data and visualization of relative abundances
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">13:00-14:00</td>
    <td colspan="4" align="center"><i>Lunch Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">14:00-14:30</td>
    <td colspan="4" align="center">Questions and answers</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">14:30-15:30</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Theoretical notes on the concept of Diversity and Diversity measures<br>
      Data normalization for Rarefaction and CLR<br>
       - Dimensional reduction approaches (PCoA/PCA) and permANOVA tests<br>
      Statistical tests on alpha diversity and beta diversity metrics<br>
      Differential Abundance Analysis
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">15:30-17:00</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">
      - Rarefaction and diversity metrics<br>
      - Statistical Comparison<br>
      - Differential Abundance Analysis
    </td>
  </tr>
  <tr>
    <td colspan="4" align="center"><h3>Day 3 - Artificial Intelligence and the Microbiome</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">Time</td>
    <td height="50">Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">09:00-10:30</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Fundamentals of Machine Learning and AI<br>
      Introduction to Python for microbiome data analysis<br>
      Types of Machine Learning (Supervised vs Unsupervised)<br>
      Microbiome data and their specificity<br>
       - Compositional data and log-ratio transformations (clr, ilr, alr)<br>
       - Sparsity and over-dispersion problems<br>
       - Need for normalization and scaling<br>
      Dimensionality reduction<br>
       - PCA, t-SNE, UMAP (advantages and limitations in microbiomic analysis)<br>
       - Considerations on normalization, data balancing and cross-validation
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">10:30-11:00</td>
    <td colspan="4" align="center"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">11:00-12:30</td>
    <td height="50">Exercise</td>
    <td height="50" align="left">
      Data preprocessing and preparation for ML<br>
       - Data import and visualization<br>
       - Transformations for compositional data<br>
       - Normalization and management of relative abundances<br>
       - Implementation of PCA and t-SNE to explore the structure of microbiome data
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">12:30-13:30</td>
    <td colspan="4" align="center"><i>Lunch Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">13:30-15:00</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Machine Learning Models and Interpretability<br>
      Machine Learning for the Microbiome<br>
       - Classification vs Regression<br>
       - Common algorithms: Logistic Regression, Decision Trees, Random Forest, SVM, Gradient Boosting<br>
       - Performance evaluation<br>
       - Accuracy, Precision, Recall, F1-score, AUC-ROC<br>
       - Confusion Matrix and importance of the choice of metric<br>
       - Explainable AI (XAI) and model interpretability <br>
       - (SHAP and LIME) for feature interpretation<br>
       - Importance of features and their role in prediction<br>
       - XAI applications in the microbiome: identification of relevant microbial markers
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">15:00-15:30</td>
    <td colspan="4" align="center"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">15:30-17:00</td>
    <td height="50">Exercise</td>
    <td height="50" align="left">
      Construction and interpretation of an ML model on metagenomic data<br>
       - Training a classification model on abundance data<br>
       - Hyperparameter optimization with cross-validation<br>
       - Interpretation of results with SHAP<br>
       - Discussion of the results and comparison with traditional methods
    </td>
  </tr>
  <tr>
    <td colspan="4" align="center"><h3>Day 4 - Analysis of the taxonomic and functional composition of the microbiome through shotgun metagenomics</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">09:00-10:30</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Sequencing technologies<br>
      Brief introduction to the main analysis techniques:<br>
       - Taxonomic profiling<br>
       - Functional profiling<br>
       - Metagenome assembly and binning<br>
      Computational tools for metagenomics<br>
       - operating systems<br>
       - hardware requests<br>
       - software tools (nextflow-docker, etc)<br></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">10:30-11:00</td>
    <td colspan="4" align="center"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">11:00-12:30</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Taxonomic and functional profiling using shotgun data<br>
      Raw data preprocessing: read filtering and host elimination<br>
      Taxonomic profiling:<br>
       - MetaPhlAn<br>
       - Kraken2/Bracken<br>
      Functional profiling: HUMANN<br>
      Taxonomic profiling beyond the species level: <br>
      Strain level analysis<br>
       - The species concept in bacteria<br>
       - Genomic variability within the species: strain, genome, pangenome <br>
       - Strain level profiling: StrainPhlAn
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">12:30-13:30</td>
    <td colspan="4" align="center"><i>Lunch Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">13:30-15:00</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">
       - MetaPhlAn<br>
       - Kraken2/Bracken<br>
       - HUMANN
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">15:00-15:30</td>
    <td colspan="4" align="center"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">15:30-16:00</td>
    <td height="50">Hands-On</td>
    <td height="50" align="left">StrainPhlAn</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">16:00-17:00</td>
    <td height="50">Lecture</td>
    <td height="50" align="left">
      Metagenome assembly and binning<br>
      - Binning<br>
      - Quality measures for MAGs<br>
      - Dereplication of MAGS<br>
      - Taxonomic classification of MAGs<br>
         - Hands On kMetaShot<br>
      - Functional annotation of MAGs
    </td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">20:30</td>
    <td colspan="4" align="center"><i>Social Dinner</i></td>
  </tr>
  <tr>
    <td colspan="4" align="center"><h3>Day 5 - Case Studies</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">Time</td>
    <td height="50">Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">09:00-10:00</td>
    <td height="50">Lecture</td>
    <td height="50">The human microbiome and the Holobiont theory of evolution<br>Duccio Cavalieri, University of Florence</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">10:00-11:00</td>
    <td height="50">Lecture</td>
    <td height="50">The Food Microbiome<br> Eugenio Parente, University of Basilicata</td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">11:00-11:30</td>
    <td colspan="4" align="center"><i>Coffee Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align="center">11:30-12:30</td>
    <td height="50">Lecture</td>
    <td height="50">
      Living with microbes: The microbiome of the built environment and its implications in human wellbeing<br>
      Antonella Bruno, University of Milano Bicocca
    </td>
  </tr>
</table>


