---
title: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
event_type: "ELIXIR Training Course"
type: training_courses
description: 25-28 November 2025 <br> Roma, Italy
event:
  - name: "scRNA-seq Workshop. An introduction to single cell RNA-seq data analysis"
    organisers: R. Calogero <br> L. Le Pera <br> 
    venue: Rome, Italy
    event_date: 2025/11/25-28
    

short_description: "This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package."
teachers: [Luca Alessandrì, Raffaele Calogero]
keywords: "scRNA-seq, rCASC package, clustering methods, Genomics, Bioinformatics"
courseMode: onsite
addressLocality: Rome
addressCountry: Italy
startDate: "2025/11/25"
endDate: "2025/11/28"
duration: P3DT7H
inLanguage: en-EN 
---


![Funding_Rome_technopole_logo](images/2025_events/barra_loghi_rome_technopole_Bianco.jpg){: height="1200px" width="1200px"}

![ISS_Logo](images/2025_events/Logo_ISS.png){: height="150px" width="150px"}   ![IIB_ELIXIR_LOGO](images/2025_events/IIB_ELIXIR_col.png){: height="300px" width="300px"}

<br>
<br>

## Course Description

This course offers a comprehensive introduction to single-cell RNA sequencing (scRNA-seq) data analysis using open-source tools, with a focus on the rCASC package [https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC). Participants will delve into experimental design principles, quality control of single-cell sequencing outputs, and data reduction and clustering techniques to identify cell subpopulations. The course also provides an overview of the computational theory behind scRNA-seq tools, empowering participants to make informed methodological decisions during analysis. Hands-on sessions will allow attendees to work with real datasets and integrate results to extract meaningful biological insights. 
<br>
This course is organised by Spoke 3 within the framework of the [“Rome Technopole Innovation Ecosystem” project](https://www.rometechnopole.it/).
<br>


## Important Dates

- Deadline for applications: **20 Nov 2025**
- Course date: **25-28 November 2025**
<br>

### Venue

Sapienza Università di Roma, Dep. of "Chimica e Tecnologie del Farmaco", P.le Aldo Moro, 5 - 00185 Roma <br>
Aula D - Informatica - II Piano Edifico CU019 
[google link](https://maps.app.goo.gl/v7jWDZNZygiK6pqP7)

<br>

### Fee 

The course is free of charge. Participants are expected to cover their own travel and accomodation expenses (if any).
<br>

### Selection  

A maximum of **15 participants** will be selected on a first-come, first-served basis, taking into account the applicant’s background and motivation as described in the application form. Selected participants will be notified progressively as applications are reviewed. If the number of applications exceeds the maximum capacity, a waiting list will be created. Only applicants who receive a confirmation email will be officially admitted to the course.<br>

### Instructors

- Raffaele Calogero, University of Turin, Italy  
- Luca Alessandrì, University of Turin, Italy
- Valentina Libri, Istituto Superiore di Sanità (ISS) - FAST, Rome, Italy
<br>

### Scientific/Organising committee

- Raffaele Calogero, University of Turin, Italy (ELIXIR-IT)
- Loredana Le Pera, Istituto Superiore di Sanità (ISS) - FAST, Rome, Italy
- Allegra Via, Sapienza University of Rome, Italy (ELIXIR-IT Training Platform)
- Rino Ragno, Sapienza University of Rome, Italy (ELIXIR-IT LTeC)
- Giacinto Donvito, INFN, Bari, Italy (ELIXIR-IT Compute Platform)
<br>

### Organising secretary
  
- Gianmarco Pascarella - CNR, Italy (ELIXIR-IT Training Platform)
- Irene Artuso, Istituto Superiore di Sanità (ISS), Rome, Italy (ELIXIR-IT Training Platform)
<br>

## Contact 

For any queries, please contact the Local Organisers at: [elixir.ita.training@gmail.com](mailto:elixir.ita.training@gmail.com) 

<br>

### Target audience

This course is aimed at life science researchers, students, principal investigators (PIs), and industry professionals interested in analysing single-cell RNA-seq data. No prior knowledge of statistics or advanced computing is required. However, **a basic understanding of R programming** and **bioinformatics concepts** is preferred. **Prior knowledge of single-cell sequencing technologies** is also required.
<br>

### Aims of the workshop 

- Introduce the principles and importance of designing effective scRNA-seq experiments.   
- Guide participants through quality control, data reduction, and clustering methods for scRNA-seq data.   
- Provide hands-on experience to develop practical analysis skills and extract meaningful biological insights from scRNA-seq data.
<br>

### Resources and tools covered

rCASC package for scRNA-seq analysis ([https://github.com/kendomaniac/rCASC](https://github.com/kendomaniac/rCASC))
<br>

### Learning Outcomes

By the end of this course, participants will be able to:
* Describe the importance of experimental design to ask sensible biological questions at single-cell level.  
* Assess the quality of your data.  
* Describe limitations and strengths of clustering in scRNA-seq.  
* Identify genes driving cluster formation.  
* Annotate cell types  
* Perform batch correction and experiment integration on scRNA-seq data.
<br>

### Course prerequisites

Before starting the course, participants are requested to complete a short introductory R programming module (~4 hours). The video lessons and PDFs with exercises are available at [https://bit.ly/47XmDwx](https://bit.ly/47XmDwx).
Participants are also expected to bring their own laptop for the hands-on sessions.
<br>

### Learning assessment

At the end of the course, all participants will be required to complete an assessment test. Additionally, a course evaluation questionnaire will be administered to collect participants’ feedback.

### Certificates

At the end of the event, participants who request it will receive a certificate of attendance. A **certificate of participation**, including the total number of training hours, will be issued to participants who have attended at least 80% of the course and achieved a minimum score of 75% on the final assessment.

## Registration

### [Application Form](https://forms.gle/rjqURTQumDNysC9p7)
<br>
<br>

## Preliminary Programme

<table border="1" width="700">
<tr>
    <td colspan="3" align=center><h3>25 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>Time</td>
    <td height="50" align=center>Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
 <tr>
    <td height="50" width="100" align=center>14:30-16:30</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Welcome and introduction to the workshop<br>Lecture: Experimental overview of single-cell RNA-seq<br>Setting up the working environment</td>
  </tr>
 
 <tr>
    <td colspan="3" align=center><h3>26 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>Time</td>
    <td height="50" align=center>Learning Experience</td>
    <td height="50">Topic</td>
  </tr>
    <tr>
    <td height="50" width="100" align=center>09:30-11:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Experimental design <br>Reproducibility<br>Dense and sparse matrices</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>11:00-11:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-12:30</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Data structure 10xgenomics, visium, curio bioscience</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=center><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-15:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">QC<br>Data imputation/normalisation</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:00-15:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:30-16:30</td>
    <td height="50" align=center>Exercise</td>
    <td height="50">Exercises</td>
  </tr>
  <tr>
    <td colspan="3" align=center><h3>27 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-11:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Dimensionality reduction<br>Clustering</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>11:00-11:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>11:30-12:30</td>
    <td height="50" align=center>Exercise</td>
    <td height="50">Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=center><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-15:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Cluster's specific genes extraction</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:00-15:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:30-16:30</td>
    <td height="50" align=center>Exercise</td>
    <td height="50">Exercises</td>
  </tr>
  <tr>
    <td colspan="3" align=center><h3>28 November</h3></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>09:30-11:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Cell type annotation</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>11:00-11:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>11:30-12:30</td>
    <td height="50" align=center>Exercise</td>
    <td height="50">Exercises</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>12:30-13:30</td>
    <td colspan="3" height="50" align=center><i>Lunch break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>13:30-15:00</td>
    <td height="50" align=center>Lecture</td>
    <td height="50">Data integration</td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:00-15:30</td>
    <td colspan="3" height="50" align=center><i>Break</i></td>
  </tr>
  <tr>
    <td height="50" width="100" align=center>15:30-16:30</td>
    <td height="50" align=center>Exercise</td>
    <td height="50">Exercises</td>
  </tr>
</table>


![Single-Cell_Omics_Logo](images/2025_events/sco.png){: height="200px" width="200px"}![BITS_Logo](images/2025_events/BITSLogo.jpg){: height="200px" width="200px"}![Infolife](images/2025_events/Logo_infolife.jpg){: height="200px" width="200px"}
